app/src/main/java/com/example/memeapp/presentation/activity/MainActivity.kt

```diff
@@ -36,4 +39,8 @@ class MainActivity : AppCompatActivity(), ItemClicked {
     override fun memeCardClicked(memeId: Int) {
         presenter.onMemeClicked(memeId)
     }
+
+    private fun loadMemesFromLocalDB() {
```

Comment:
rename this method pls. It can be called setListeners()

Reply:
Done thx